### PR TITLE
Prs7597 nob corr mk 20.09.2024

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: PRS7001-8000/PRS7597Nob.xml [hunk 1]
```diff
@@ -32,26 +32,38 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-               </persName>
-               <occupation> abbot of Rema </occupation>
+               <persName xml:lang="gez" xml:id="n1">ኖብ፡</persName>
+               <persName xml:lang="gez" type="normalized" corresp="#n1">Nob</persName>
+               <floruit>He was the brother of ʾaṣe <persName ref="PRS10342Yeshaq">Yǝsḥāq</persName> and spiritual son of <persName ref="PRS10253Yasay">Yāsāy</persName>.
+             He is credited as the founder of the monastic community of <ref type="ins" corresp="INS0359Rema">Remā</ref>.</floruit>
             </person>
+            <listRelation>
+               <relation name="snap:BrotherOf" active="PRS7597Nob" passive="PRS10342Yeshaq"/>
+           </listRelation>
```
Comment (eu-genia) [inline PRS7001-8000/PRS7597Nob.xml:59]: you can also add already here the relation formerlyListedAs to PRS14216NobWaldaYasey (and possibly also delete PRS14216NobWaldaYasey as suggested in https://github.com/BetaMasaheft/Documentation/issues/2650) to save the number of branches/pull requests

Comment (eu-genia) [reply PRS7001-8000/PRS7597Nob.xml:59]: Possibly the whole string Nob Walda Yasay can be added also as n2

**Review by eu-genia — APPROVED** [review]

File: PRS7001-8000/PRS7597Nob.xml [hunk 2]
```diff
@@ -54,11 +54,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-               <floruit>He was the brother of ʾaṣe <persName ref="PRS10342Yeshaq">Yǝsḥāq</persName> and spiritual son of <persName ref="PRS10253Yasay">Yāsāy</persName>.
-             He is credited as the founder of the monastic community of <ref type="ins" corresp="INS0359Rema">Remā</ref>.</floruit>
+               <persName xml:lang="gez" type="normalized"><roleName type="title">ʾabbā</roleName></persName>
+               <occupation type="ecclesiastic">He is credited as the founder of the monastic community of 
+                  <ref type="ins" corresp="INS0359Rema">Remā</ref>.</occupation>
+               <floruit notBefore="1350" notAfter="1450">Mid 14th to mid 15th century</floruit>
             </person>
             <listRelation>
-               <relation name="snap:BrotherOf" active="PRS7597Nob" passive="PRS10342Yeshaq"/>
+               <relation name="snap:BrotherOf" active="PRS7597Nob" passive="PRS10342Yeshaq"><desc>He was the brother
```
Comment (eu-genia) [inline PRS7001-8000/PRS7597Nob.xml:63]: note that the descriptions of relations are only visible from the tab relations and not from the main record; i think the spiritual relation is too important to be "hidden" this way

Comment (CarstenHoffmannMarburg) [reply PRS7001-8000/PRS7597Nob.xml:63]: OK. I will move it back to floruit. Probably, it is useful to have it both in floruit and in relation.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**